### PR TITLE
Add org backend model

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: dash/orgs/views.py [hunk 1]
```diff
@@ -121,62 +121,10 @@ def get_object(self, *args, **kwargs):
 
 
 class OrgCRUDL(SmartCRUDL):
-    actions = ('create', 'list', 'update', 'choose', 'home', 'edit', 'tokens',
+    actions = ('create', 'list', 'update', 'choose', 'home', 'edit',
                'manage_accounts', 'create_login', 'join', 'chooser')
     model = Org
 
-    class Tokens(InferOrgMixin, OrgPermsMixin, SmartUpdateView):
```
Comment (norkans7) [inline dash/orgs/views.py]: no longer needed we manage the api token on the OrgBackendCRUDL views

File: dash/orgs/views.py [hunk 2]
```diff
@@ -749,3 +693,35 @@ def lookup_field_link(self, context, field, obj):
                 return reverse('orgs.org_update', args=[obj.org_id])
             else:
                 return super(TaskCRUDL.List, self).lookup_field_link(context, field, obj)
+
+
+class OrgBackendForm(forms.ModelForm):
+    backend_type = forms.ChoiceField(choices=settings.DATA_API_BACKEND_TYPES)
```
Comment (norkans7) [inline dash/orgs/views.py]: settings for each projects to list its backend types

File: dash/orgs/models.py [hunk 3]
```diff
@@ -345,3 +353,19 @@ def get_time_taken(self):
...
+
+
+class OrgBackend(SmartModel):
+    org = models.ForeignKey(Org, related_name='backends')
+
+    slug = models.CharField(max_length=16)
+
+    backend_type = models.CharField(max_length=256, null=True, blank=True)
+
+    host = models.CharField(max_length=128, null=True, blank=True)
```
Comment (rowanseymour) [inline dash/orgs/models.py]: can you have a backend without a type / host / token? seems like some of these shouldn't be nullable

**Comment by norkans7** [comment]
@rowanseymour updated to make all fields not null

**Pull request merged**